### DISCUSSION TITLE
Twitch notification / tags with macro ?

### DISCUSSION THREAD

**Discussion by seigneurao** [WarmUpTill/SceneSwitcher#1181] [Ideas]
Hello.

First of all, thanks for this very useful tool, and your work.
I have a question/request : would it be possible to add macro actions for Twitch, allowing to set the Notification message (sent to followers at the beginning of the stream) and/or tags, the same way we can set the Title and Category.
Or is it just not feasible, due to Twitch API limitations of some sort ?

Thanks in advance !

**Comment by WarmUpTill** [comment]
Unfortunately, I believe that is simply not possible, because the Twitch API does not offer any such functionality.
The addition of such an API call was requested quite a while ago, but Twitch never implemented it as far as I can tell:
https://twitch.uservoice.com/forums/310213-developers/suggestions/39703660-live-notification
(Please don't click any of the spam links in the comment section of the uservoice website)

**Reply by seigneurao** [reply]
Thanks for the quick and clear answer !

**Comment by finalmakerr** [comment]
There is a cool workaround to make it works by having a chat overlay browser source in OBS.
Add a sound to that overlay.
Make a condition that trigger by certain volume of that source.

I can provide more [info](https://discord.gg/F3CCAQpZqZ) if needed.
